src/main/java/com/juniorstart/juniorstart/helpers/getLogedUserId.java

```diff
@@ -0,0 +1,13 @@
+package com.juniorstart.juniorstart.helpers;
+
+import com.juniorstart.juniorstart.model.User;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+public class getLogedUserId {
+
+    Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
```

Comment:
Już jest adnotaja @CurrentUser, która zwraca użytkownika zalogowanego

Reply:
Poprawię to w następnym branchu. Powyższa funkcjonalność nie jest w tym wykorzystywana. Sorry za śmieciowy kod.